libse/SubtitleFormats/UnknownSubtitle4.cs

```diff
@@ -115,21 +115,7 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                     {
                         if (line.Length > 0)
                         {
-                            string text = line.Replace("|", Environment.NewLine);
-                            text = line.Replace("[br]", Environment.NewLine);
-                            text = line.Replace("<br>", Environment.NewLine);
-                            text = line.Replace("<br />", Environment.NewLine);

```

Comment:
BUG! Note that line is being used again.
`text = line.Replace("[br]", Environment.NewLine);` =>  `text = text.Replace("[br]", Environment.NewLine);`